Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -400,6 +400,14 @@
     accentVColor: "#8900c9"
   - type: Icon
     state: pda-miner
+  - type: CartridgeLoader
+    uiKey: enum.PdaUiKey.Key
+    notificationsEnabled: false
```

Comment:
Why are you disabling notifications by default here?

Reply:
Great question! Left over from debugging. Easy fix.